python/lsst/ts/scheduler/utils/types.py

```diff
@@ -0,0 +1,7 @@
+from lsst.ts.salobj import DefaultingValidator
+
+__all__ = [
+    "ScriptsConfigValidator",
+]
+
+ScriptsConfigValidator = dict[(str, bool), DefaultingValidator]
```

Comment:
I find it very confusing that this is a dict, is used as a dict and referred to as a dict without having `Dict` in the name. Please consider renaming it to `ScriptsConfigValidatorDict` or something similar.